docs/modules/sql/pages/create-mapping.adoc

```diff
@@ -94,8 +94,37 @@ OPTIONS (
     'valueFormat'='int');
 ----
 
+|`data_connection_name`
+|An existing data connection that is used to connect to the external system. 
+|
+[source,sql]
+----
+CREATE MAPPING mytransactions
+DATA CONNECTION myJDBCconnection
+OPTIONS (
+    'keyFormat'='varchar',
+    'valueFormat'='varchar');
```

Comment:
Wrong sample - keyFormat and valueFormat are not valid for jdbc connection, rather use myKafkaConnection

Reply:
I've updated both of the examples that use a data connection.